lib/internal/test_runner/test.js

```diff
@@ -666,6 +666,15 @@ class Test extends AsyncResource {
       this.reporter.diagnostic(this.indent, this.diagnostics[i]);
     }
   }
+
+  reportSubtest() {
+    if (this.#reportedSubtest || !this.parent) {
```

Comment:
To be consistent with the other `.parent` checks
```suggestion
    if (this.#reportedSubtest || this.parent !== null) {
```